server/src/config.rs

```diff
@@ -2,13 +2,23 @@
 
 //! Server configs
 
+use std::collections::HashMap;
+
 use analytic_engine;
-use cluster::config::ClusterConfig;
-use meta_client::MetaClientConfig;
+use cluster::{config::ClusterConfig, SchemaConfig};
+use common_types::schema::TIMESTAMP_COLUMN;
 use serde_derive::Deserialize;
+use table_engine::ANALYTIC_ENGINE_TYPE;
 
-use crate::router::RuleList;
+use crate::route::rule_based::{ClusterView, RuleList, ShardView};
 
+/// The deployment mode decides how to start up the CeresDB.
```

Comment:
```suggestion
/// The deployment mode decides how to start the CeresDB.
```

"start up" is means another things